models/contributor.py

```diff
@@ -157,3 +157,48 @@ async def post_to_twitter(self) -> None:
             file=self.image_bytes,
             filename="contributor.png",
         )
+
+    async def get_issue_count(self) -> Any:
```

Comment:
I think we can get both the PR count and issue count from the same API call and loops
..
Note: GitHub's REST API considers every pull request an issue, but not every issue is a pull request. For this reason, "Issues" endpoints may return both issues and pull requests in the response. You can identify pull requests by the pull_request key. Be aware that the id of a pull request returned from "Issues" endpoints will be an issue id. To find out the pull request id, use the "[List pull requests](https://docs.github.com/rest/reference/pulls#list-pull-requests)" endpoint.